packages/editor/src/dataviews/types.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import type { AnyItem } from '@wordpress/dataviews';
+
+type PostStatus =
+	| 'published'
+	| 'draft'
+	| 'pending'
+	| 'private'
+	| 'future'
+	| 'auto-draft'
+	| 'trash';
+
+export interface Post extends AnyItem {
```

Comment:
Ideally these "Model" types should be in core-data but it's fine for now here. I think ultimately the whole "dataviews" folder here will move to "core-data".